verify-sna/functions/remove-old-verifications.js

```diff
@@ -0,0 +1,145 @@
+/**
```

Comment:
Sir, question. When is this guy called in the process?

Reply:
That function isn't still being called yet. However the idea is to call that function from the start-verify and check-verify endpoints, or even from the front-end while doing the polling action (I already I have the polling implemented but it is not included in this branch)